src/toffy/mph_comp.py

```diff
@@ -24,7 +24,8 @@ def get_estimated_time(bin_file_dir, fov):
     io_utils.validate_paths(bin_file_dir)
 
     # get fov json file in bin_file_path
-    json_file = io_utils.list_files(bin_file_dir, fov + ".json")
+    json_file = sorted(io_utils.list_files(bin_file_dir, fov, exact_match=True))
+    json_file = list(filter(lambda f: ".json" in f, json_file))
```

Comment:
I think for this case, since we always know we will want the file name `fov + '.json'`, I think we can simply define json_file to be that. It's unnecessary to list the files (returns ['fov-1-scan-1.json', 'fov-1-scan-1.bin']), then just take take the json file only anyways. Then we will ignore the -processing.json without having to specify exact_match.

Reply:
It won't work if you set the second argument (substr(s)) to `fov-1-scan-1.json` because of the way `list_files` trims the extension for each file listed in `bin_file_dir`, but then compares it to `fov-1-scan-1.json`. So since there's a `.bin` and a `.json` file, you'd be making two comparisons of `fov-1-scan-1` to `fov-1-scan-1.json`, none of which will return True. This caused the previous tests to hang.